pdc_client/__init__.py

```diff
@@ -203,10 +205,21 @@ def get_paged(self, res, **kwargs):
         """
         if self.page_size is not None:
             kwargs['page_size'] = self.page_size
-
+            page_size = self.page_size
             if self.page_size <= 0:
                 # If page_size <= 0, pagination will be disable.
                 return res(**kwargs)
+        else:
+            page_size = 20
+        if self.page is not None:
+            allinfo = res(page=self.page, **kwargs)
+            all_count = int(allinfo["count"])
+            if all_count % page_size > 0:
+                total = (all_count / page_size) + 1
+            else:
+                total = all_count / page_size
+            print("Total %d pages, here is the %d page, the count is %s." % (total, self.page, all_count))
```

Comment:
Good catch, if JSON mode, should not print this line.

Currently, it is on the first line, I think it's okay, if I put it in the each print table functions, that means the return value of the get_paged should change, and all the plugins will insert a print line, I also don't think that is a good idea. Which is the better way?

For me, think this get_pages function is just used for our codes, so it's okay to write the print line here. 
Because there is another issue to improve this PDCClient about the paging, I think I can give another patch for common use.

I will modify codes and just let this line print when it is not json output. What'[s your thinking?